content/general/concepts/augmented-reality/augmented-reality.md

```diff
@@ -19,7 +19,7 @@ An augogram is the computer generated image that is used to create an Augmented
 
 ## Augmented Reality vs. Virtual Reality
 
-The difference between augemented reality (AR) and virtual reality (VR) is that augmented reality utilizes the current physical setting of the user while introducing a layer of synthetic digital, sonic or somatic presentation to enhance the user experience whereas VR involves use of a headset to facilitate an experience rooted in the graphic and sonic presentation within the VR setup as opposed to using the individuals more broad physical surroundings for interaction.
+The difference between augmented reality (AR) and virtual reality (VR) is that augmented reality utilizes the current physical setting of the user while introducing a layer of synthetic digital, sonic or somatic presentation to enhance the user experience whereas VR involves use of a headset to facilitate an experience rooted in the graphic and sonic presentation within the VR setup as opposed to using the individuals more broad physical surroundings for interaction.
```

Comment:
This sentence is quite long! Any chance it could be broken up?

Reply:
Hi @HishamT . I break this sentence into two.